docs/fundamentals/code-analysis/style-rules/ide0110.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "IDE0110: Discard can be removed"
```

Comment:
I think we should keep the title consistent with other unnecessary code rules, i.e. `Remove unnecessary discard`
```suggestion
title: "IDE0110: Remove unnecessary discard"
```

Reply:
Yes, we have evidence that users search on the code rule text.